test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -84,14 +92,17 @@ private void VerifySdkImage_NewRestoreRun(ImageDescriptor imageDescriptor, strin
 
             string buildArgs = GetBuildArgs(args.ToArray());
             string sdkImage = GetDotNetImage(
-                imageDescriptor.SdkVersion, DotNetImageType.SDK, imageDescriptor.OsVariant);
+                imageVersion: imageDescriptor.DotNetCoreVersion, imageType: DotNetImageType.SDK, osVariant: imageDescriptor.SdkOsVariant, isArm: false);
```

Comment:
Line length is too long.  I usually try to keep lines in that less than 120 char ballpark.  Makes it easier to view especially in GitHub.  Comment applies to other changes as well.

Reply:
Agree. Do you do this manually? Is there an styling rule or extension the make this semi-automated.

Reply:
I found one extension - https://marketplace.visualstudio.com/items?itemName=PaulHarrington.EditorGuidelines
Guidelines here don't give an option for line length -
 https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

Reply:
I handle this manually.  It is not an easy thing to automate as there are so many cases and it seems everyone has a different preference.  For method calls, I will first break the parameters onto a single separate line.  If even that is too long, then I will put each parameter on it's own line.

For example, if a call exceeds 120 chars
```
string sdkImage = GetDotNetImage(imageDescriptor.SdkVersion, DotNetImageType.SDK, imageDescriptor.OsVariant);
```

I will first try this
```
string sdkImage = GetDotNetImage(
    imageDescriptor.SdkVersion, DotNetImageType.SDK, imageDescriptor.OsVariant);
```

If that is still to long I will use this
```
string sdkImage = GetDotNetImage(
    imageDescriptor.SdkVersion,
    DotNetImageType.SDK,
    imageDescriptor.OsVariant);
```

Reply:
Thanks Michael. Updated accordingly.